### PR TITLE
DOC-1760: Document `transparent` prop for iframe UI component

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: DOC-1760

Description of Changes:
* Document new `transparent` property for iframe UI component

Pre-checks:
- [X] Branch prefixed with `feature/6/` or `hotfix/6/`
- [X] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [X] Files has been included where required (if applicable)
- [X] Files removed have been deleted, not just excluded from the build (if applicable)
- [X] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by lnewson — APPROVED** [review]

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/dialog-components.adoc [hunk 1]
```diff
@@ -330,6 +330,7 @@ NOTE: To replace the entire dialog body with an iframe that loads its content fr
 |name |string |required |A identifier for the button.
 |label |string |optional |String to use for the iframe's `+title+` attribute.
 |sandboxed |boolean |optional |default: `+true+` - When true, sets the iframe's `+sandbox+` attribute to `+"allow-scripts allow-same-origin"+`.
+|transparent |boolean |optional |default: `+true+` - When true, sets the iframe's background to transparent instead of opaque.
```
Comment (lnewson) [inline modules/ROOT/pages/dialog-components.adoc:333]: This needs the `NOTE: This feature is only available for TinyMCE 6.1 and later.` warning as well I think?

Comment (ltrouton) [reply modules/ROOT/pages/dialog-components.adoc:333]: Yeah, I was just working on adding it 👍

Comment (ltrouton) [reply modules/ROOT/pages/dialog-components.adoc]: Done

Comment (lnewson) [reply modules/ROOT/pages/dialog-components.adoc]: This should be using the partial shouldn't it?

Comment (ltrouton) [reply modules/ROOT/pages/dialog-components.adoc:333]: Yep, fixed.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**